superset/contrib/docker/.env

```diff
@@ -1,17 +0,0 @@
-#
```

Comment:
why do you need to delete it?

Reply:
We don't rely on the contents of `.env` for a normal installation of `sandbox-ce`, so I wanted to notify this somehow here. If you add variables here, they will not have an effect on our final distribution.